src/app/pages/pupil-profile/pupil-profile.component.html

```diff
@@ -0,0 +1,5 @@
+<yps-profile-card [user]="userService.user" ></yps-profile-card>
+<div class="">
+    <yps-pupil-event-card></yps-pupil-event-card>
+
+</div>
```

Comment:
What is this? I think you don't need this 'div'.

Reply:
Fixed!